src/constants.ts

```diff
@@ -6,3 +6,26 @@
 export const CALENDAR_APP_ID = 'carbonio-calendars-ui';
 
 export const CALENDAR_ROUTE = 'calendars';
+
+export const settingsSubSections = [
+	{
+		label: 'label.general',
+		fallback: 'General'
```

Comment:
you can add some related key here and use to fetch the correct label instead of using index